emperor/support_files/js/color-editor.js

```diff
@@ -9,21 +14,21 @@ function($, _, DecompositionView, ViewControllers, spectrum) {
 
   /**
    *
-   * @name ColorEditor
+   * @class ColorEditor
    *
-   * @class This class represents a color editor defined by the SlickGrid
+   * This class represents a color editor defined by the SlickGrid
    * project.
    *
-   * @property {$input} object where the Spectrum color picker lives.
-   * @property {defaultValue} initial value of the cell being edited.
-   *
    * Note, this object is heavily based on classes in slick.editors.js and in
-   * the documentation that can be found here:
-   *    https://github.com/mleibman/SlickGrid/wiki/Writing-custom-cell-editors
+   * the documentation that can be found [here](https://github.com/mleibman/
```

Comment:
I tried using the `@link` directive, but since the URL spans multiple
lines, the whitespace messes with the link and leads to a 404. This
problem doesn't seem to happen with the markdown syntax though.

On (May-25-16|22:59), Joshua Shorenstein wrote:
> >     * Note, this object is heavily based on classes in slick.editors.js and in
> > -   * the documentation that can be found here:
> > -   *    https://github.com/mleibman/SlickGrid/wiki/Writing-custom-cell-editors
> > +   * the documentation that can be found [here](https://github.com/mleibman/
> 
> To be more "JSDoc-y", you can change
> (https://github.com/mleibman/SlickGrid/wiki/Writing-custom-cell-editors)
>  to
> {@link https://github.com/mleibman/SlickGrid/wiki/Writing-custom-cell-editors}
> and it will still render as a link.
> 
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/497/files/33689131f40376307cf493cd8c31f39c59647365#r64694961